### PR TITLE
add support for pravega-0.6.0 / 0.13-W13

### PR DESCRIPTION
**Pull request by damienmas**
this PR is for adding support with the new pravega version and tls://controller.fqdn:433
there is no need to download keycloak manually or build pravega and flink-connector

modified:   README.md
modified:   gradle.properties
modified:   camera-recorder/build.gradle
modified:   flinkprocessor/build.gradle
modified:   video-player/build.gradle

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,21 +8,28 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### dependencies
+# For Nautilus 0-13-w11
```
Comment (claudiofahey) [inline gradle.properties]: Please keep this list in alphabetical order.

File: gradle.properties [hunk 2]
```diff
@@ -8,21 +8,28 @@
...
 flinkVersion=1.7.2
+flinkShortVersion=1.7
 junitVersion=4.12
 logbackVersion=1.2.3
 logstashLogbackEncoderVersion=4.11
 nettyBoringSSLVersion=2.0.17.Final
 nettyVersion=4.1.30.Final
 # Below is for Nautilus 0.12.0
-pravegaCredentialsVersion=0.6.0-2345.298015f-0.12.0-W5-001.4e5c9a1
+# pravegaCredentialsVersion=0.6.0-2345.298015f-0.12.0-W5-001.4e5c9a1
```
Comment (claudiofahey) [inline gradle.properties]: Go ahead and remove this line and other obsolete lines.

**Review by claudiofahey — CHANGES_REQUESTED** [review]
Looks good. Just minor changes.

File: gradle.properties (hunk 1 above)
Comment (damienmas) [reply gradle.properties]: done

File: gradle.properties (hunk 2 above)
Comment (damienmas) [reply gradle.properties]: done , please review

**Review by claudiofahey — APPROVED** [review]

**Comment by claudiofahey** [comment]
These changes have been included in PR #8.

**Pull request closed without merging**